app/views/pages/companyDetails/tradingName.scala.html

```diff
@@ -39,31 +39,32 @@
 @main_template(title = title(Messages("pages.tradingName.description"), tradingNameForm)) {
 
     @payeErrorSummary(
-        Messages("app.common.errorSummaryLabel"), tradingNameForm, dataJourney=Some("TradingName")
+        Messages("app.common.errorSummaryLabel"), tradingNameForm, dataJourney = Some("TradingName")
     )
 
     <h1 class="form-title heading-large" id="pageHeading">@Messages("pages.tradingName.description")</h1>
 
 
-     <p id="lead-paragraph">@messages("pages.tradingName.lede", companyName)</p>
+    <p id="lead-paragraph">@messages("pages.tradingName.lede", companyName)</p>
 
-     @hiddenDetails("tradingNameRules", Messages("pages.tradingName.nameRulesSummary"), hiddenDetailsContent)
+    @hiddenDetails("tradingNameRules", Messages("pages.tradingName.nameRulesSummary"), hiddenDetailsContent)
 
     @govHelpers.form(action = controllers.userJourney.routes.CompanyDetailsController.submitTradingName) {
 
-    <br>
-    <div class="inline form-group">
+        <br>
+        <div class="inline form-group">
         @formHiddenYesNoRadio(
             tradingNameForm,
             "differentName",
             Messages("pages.tradingName.description"),
+            true,
```

Comment:
You should always provide the name the boolean for context as well as the value. From reading this file only it's not possible to tell what the boolean is for